### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by batoorsohail**
### In this Project We:

- [x] Created a table named owners with the following columns:
   - id: integer (set it as autoincremented PRIMARY KEY)
   - full_name: string
   - age: integer

- [x] Created a table named species with the following columns:
   - id: integer
   - name: string

- [x] Removed column species
- [x] Added column species_id which is a foreign key referencing species table
- [x] Added column owner_id which is a foreign key referencing the owners table
- [x] Inserted data inside owners and species Table
- [x] Modified inserted animals so it includes the species_id value
- [x] Modified inserted animals to include owner information (owner_id)
- [x] Added new queries
tables

species

fk

insertDt

speciesUpd

ownerUpd

ownerUpd2

qu1

qu2

qu3

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,4 +11,26 @@ VALUES('Charmander', '02 08 2020', -11, false, 0),
...
+      ('Digimon');
+
+-- Inserted animals so it includes the species_id value
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE name NOT LIKE '%mon';
+
+-- Inserted animals to include owner information (owner_id)
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') WHERE animals.name LIKE 'Agumon';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') WHERE animals.name IN ('Gabumon', 'Pikachu');
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE animals.name IN ('Charmander', 'Squirtle', 'Blossom');
```
Comment (BenMukebo) [inline data.sql:35]: - [ ] Great job so far 👏🏻 However you have forgotten updation one inserted animals  👇🏻 Kindly add this. ✍🏻 

image

- Your code might look like this 👇🏻 

```sql
UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
   WHERE name = 'Devimon' OR name = 'Plantmon';
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @batoorsohail,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [OPTIONAL] Kindly remove  React.js in the Client Tech Stack and Express.js in the Server Tech Stack since we're not using them in the project 👇🏻 


image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (batoorsohail) [reply data.sql:35]: Thanks @BenMukebo 🙌 
I will add them now

**Review by christianbiring1 — APPROVED** [review]
Hi @batoorsohail , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Descriptive PR `title` and `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**